### ISSUE TITLE
Crash on app open 

### ISSUE THREAD

**Issue by Lastaapps** [TylerWilliamson/QuickWeather#58]
Android 11 and 12L, both Samsung
Version 2.3.0

```
FATAL EXCEPTION: main
Process: com.ominous.quickweather, PID: 15963
java.lang.RuntimeException: Unable to resume activity {com.ominous.quickweather/com.ominous.quickweather.activity.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5433)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5472)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2498)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8855)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
	at v4.o.i(SourceFile:31)
	at y4.a.a(SourceFile:20)
	at com.ominous.quickweather.activity.MainActivity.H(SourceFile:39)
	at com.ominous.quickweather.activity.MainActivity.onResume(Unknown Source:3)
	at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1494)
	at android.app.Activity.performResume(Activity.java:8477)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5403)
```

**Comment by TylerWilliamson** [comment]
I already pushed a silent fix for this issue (version 55, still "2.3.0"), but F-Droid had already built version 54. This should be resolved the next time that F-Droid builds. As a workaround, you can clear the data for the app

**Comment by Lastaapps** [comment]
Right, thanks for you work ❤️
